docs/docs/content/number.md

```diff
@@ -22,13 +22,81 @@ the best value based on the value of other parameters. But it may be necessary
 to set it manually in situations where the data sink only accepts certain
 widths (e.g. postgres).
 
+### Defaults
+
+If no variant (such as `range`, `constant`, etc.) is specified, `number`
```

Comment:
This piece of doc doesn't really have continuity.

`defaults to: - For integer subtypes...: generating one of the...`

Basically I don't follow what is happening here :joy:

Reply:
@christoshadjiaslanis can you take a look at latest and see if this is satisfactory to you?